packages/darklang/languageTools/parser/canvas.dark

```diff
@@ -0,0 +1,171 @@
+module Darklang =
+  module LanguageTools =
+    module Parser =
+      module Canvas =
```

Comment:
I forget where we last stood here - are these used by CliScript parsing? If so, should there be a TODO to stop re-using this "canvas parsing" for cli-script parsing?

Reply:
oh or maybe this is just separately updated but no longer reused - if so, ignore.

Reply:
No, this whole module isn't used anywhere. I just kept it to use when we parse canvases with the new parser

Reply:
great, sounds good